packages/@o3r/apis-manager/schematics/ng-add/index.ts

```diff
@@ -14,17 +14,17 @@ export function ngAdd(options: NgAddSchematicsSchema): Rule {
       const { ngAddPackages, getO3rPeerDeps, applyEsLintFix, getProjectFromTree } = await import('@o3r/schematics');
       const { updateApiDependencies } = await import('../helpers/update-api-deps');
       const depsInfo = getO3rPeerDeps(path.resolve(__dirname, '..', '..', 'package.json'));
-      const rulesToExecute = [];
+      const rulesToExecute: Rule[] = [];
       const workspaceProject = getProjectFromTree(tree, options.projectName || undefined);
       if (workspaceProject.projectType === 'application') {
         rulesToExecute.push(updateApiDependencies());
       }
 
-      return chain([
+      return () => chain([
```

Comment:
Because chain()(tree, context) does not return a Rule but (Rule | ((tree: Tree, context: SchematicContext) => Promise<void | Rule | Tree | Observable<Tree>>))[]'